conf/code-lists/errors-dms-rej-list.csv

```diff
@@ -1,199 +1,199 @@
-CDS71236,Licence not valid for declared Country of Origin,
-CDS71252,Licence not valid for declared Country of Dispatch,
-CDS70758,Licence is not valid for declared Country of Destination,
-CDS77001,Quantity must be provided for Document Category,
-CDS77005,Reason for waiver must be declared for Document Category,
-CDS70761,Licence has insufficient quantity remaining to meet claim,
-CDS71216,Transaction not permitted; Licence is inactive and is no longer available for use,
-CDS71206,Licence is already exhausted,
-CDS71204,Licence is already surrendered,
-CDS77004,Reservation could not be made,
-CDS12132,DUCR format not valid,
-CDS12133,MUCR format not valid,
-CDS12134,DUCR and MUCR formats not valid,
-CDS12062,External Validation DUCR result: validation errors encountered,External ILE validation DUCR result - not OK
-CDS12044?,External Validation MUCR result: validation errors encountered,
-CDS10046,Missing document not required for requested procedure,
-CDS20002,Related movement does not match declaration,
-CDS40069,Quota closed,
-CDS40068,Additional Excise code is missing,
-CDS70711,Licence Error: Electronic licence does not exist,
-CDS70719,Licence Error: Electronic licence quoted although Document Status Code indicated paper licence,
-CDS70724,Licence Error: Licence is not valid,
-CDS70739,Licence Error: An Import licence cannot be used on an Export declaration,
-CDS70740,Licence Error: Importer/Exporter is not the owner of the licence,
-CDS70760,Licence Error: Licence is not valid for the declared goods item,
-CDS70791,Licence Error: Licence line number does not exist on the licence,
-CDS71115,Licence Error: An Export licence cannot be used on an Import declaration,
-CDS71200,Uniqueness Error: Document Codes with identical IDs are not permitted on a single goods item,
-CDS71201,Licence Error: All instances of a licence for a declaration must use a consistent Document Status Code,
-CDS71203,"Licence Error: Document Status Code ""EL"" or ""EP"" is not allowed on a Single Use licence",
-CDS71262,"Licence Error: Document Status Code ""EA"" can only be declared on Type Y declarations",
-CDS73329,Licence Error: Licence cannot be declared on declaration category/declaration type combination,
-CDS77000,Obligation Error: ID must be provided for declared Document Code,
-CDS77002,Obligation Error: Document Status Code must be provided for declared Document Code,
-CDS77003,Licence Error: DUCR must be provided for an Import declaration with a Licence,
-CDS14000,Credibility check: incredible quota value found,
\ No newline at end of file
+CDS10001,Obligation Error: Mandatory Data Element has not been provided,Obligation error: Mandatory Data Element has not been provided,
+CDS10002,Obligation Error: Data Element is not allowed,Obligation error: Data Element is not allowed,
+CDS10003,Obligation Error: Number of Occurrences exceeds what is allowed for the Data Element,Obligation error: Number of Occurrences exceeds what is allowed for the Data Element,
```

Comment:
Why is each line ending in a comma?

Reply:
because the last argument is `url`, we have only one url error right now, so needed to add the empty values for rest elements.